modules/dnn/src/dnn.cpp

```diff
@@ -1299,6 +1308,10 @@ struct Net::Impl
             }
         }
 
+        tm.stop();
+
+        layersTimings[ld.id] = tm.getTimeTicks();
```

Comment:
I think we should zero it manually for skipped layers as mentioned in documentation.